Source/dictionaries/original.json

```diff
@@ -61,6 +75,7 @@
 		"Worst" : "Vaguely Unpleasant",
 		"Wow" : "Oh GOD This is SO Boring. Please Kill Me",
 		"WOW" : "Zzzzzzzzzzz",
+        "Years" : "Minutes",
```

Comment:
Sorry about that... not sure why it didn't honor those conventions. I'll fix it and resubmit